synfig-studio/src/gui/app.cpp

```diff
@@ -1389,7 +1399,9 @@ DEFINE_ACTION("keyframe-properties", _("Properties"));
 	ACCEL("<Control><Shift>less",    "<Actions>/canvasview/seek-begin");
 	ACCEL("<Control><Shift>greater", "<Actions>/canvasview/seek-end");
 	ACCEL("<Mod1>o",                 "<Actions>/canvasview/toggle-onion-skin");
-	ACCEL("0",                       "<Actions>/canvasview/canvas-zoom-fit");
+	ACCEL("<Control>equal",									"<Actions>/canvasview/canvas-zoom-in-2"					);
+	ACCEL("<Control>minus",									"<Actions>/canvasview/canvas-zoom-out-2"					);	
+	ACCEL("<Control>0",                       "<Actions>/canvasview/canvas-zoom-fit-2");
```

Comment:
I think those vertical adjustment are better off with spaces. I made a similar review for this PR https://github.com/synfig/synfig/pull/1855#discussion_r524846015 Reason explained there

Reply:
@Keyikedalube I think I am able to understand the indentation problem, i will fix this in my PR. :)

Reply:
Squash and commit. And my review's done :)

Reply:
Resolved :)